typescript/cli/src/commands/options.ts

```diff
@@ -147,3 +147,19 @@ export const dryRunOption: Options = {
   default: false,
   alias: ['d', 'dr'],
 };
+
+export const chainCommandOption: Options = {
+  type: 'string',
+  description: 'The specific chain to perform operations with.',
+};
+
+export const addressCommandOption: Options = {
+  type: 'string',
+  description: 'An onchain address.',
+};
+
+export const concurrencyCommandOption: Options = {
```

Comment:
TODO: move this into chainmetadata

Reply:
Are you imagining this would be a property of the RPC config or something else?

Reply:
yup exactly, on the rpc config